src/components/admin/UserProfileModal/components/EditingProfileModalContent.tsx

```diff
@@ -166,49 +165,55 @@ export const EditingProfileModalContent: React.FC<CurrentUserProfileModalContent
   );
 
   return (
-    <form onSubmit={handleSubmit(onSubmit)}>
-      <ProfileModalEditBasicInfo
-        user={user}
-        containerClassName="EditingProfileModalContent__section"
-        register={register}
-        watch={watch}
-        setValue={setValue}
-        partyNameError={errors?.partyName}
-      />
-      <ProfileModalQuestions
-        editMode
-        containerClassName="EditingProfileModalContent__section"
-        questions={questions}
-        answers={answers}
-        register={register}
-      />
-      <ProfileModalEditLinks
-        containerClassName="EditingProfileModalContent__section"
-        register={register}
-        initialLinks={defaultValues.profileLinks ?? []}
-        links={links}
-        errors={errors?.profileLinks}
-        onDeleteLink={onDeleteLink}
-        onAddLink={addLinkHandler}
-      />
-      {isChangePasswordShown && (
-        <ProfileModalChangePassword
+    <div className="EditingProfileModalContent">
+      <ModalTitle>Edit profile</ModalTitle>
+
+      <form onSubmit={handleSubmit(onSubmit)}>
+        <ProfileModalEditBasicInfo
+          user={user}
+          containerClassName="EditingProfileModalContent__section"
+          register={register}
+          watch={watch}
+          setValue={setValue}
+          partyNameError={errors?.partyName}
+        />
+        <ProfileModalQuestions
+          editMode
+          containerClassName="EditingProfileModalContent__section"
+          questions={questions}
+          answers={answers}
+          register={register}
+        />
+        <ProfileModalEditLinks
           containerClassName="EditingProfileModalContent__section"
           register={register}
-          getValues={getValues}
-          errors={pick<
-            FieldErrors<UserProfileModalFormData>,
-            "oldPassword" | "newPassword" | "confirmNewPassword"
-          >(errors, ["oldPassword", "newPassword", "confirmNewPassword"])}
+          initialLinks={defaultValues.profileLinks ?? []}
+          links={links}
+          errors={errors?.profileLinks}
+          onDeleteLink={onDeleteLink}
+          onAddLink={addLinkHandler}
         />
-      )}
-      <EditProfileModalButtons
-        isChangePasswordShown={!isChangePasswordShown}
-        onChangePasswordClick={showChangePassword}
-        onCancelClick={cancelEditing}
-        isSubmitting={isSubmitting}
-        containerClassName="EditingProfileModalContent__edit-buttons"
-      />
-    </form>
+        {isChangePasswordShown && (
+          <ProfileModalChangePassword
+            containerClassName="EditingProfileModalContent__section"
+            register={register}
+            getValues={getValues}
+            errors={pick<
+              FieldErrors<UserProfileModalFormData>,
```

Comment:
This is slightly hard to read, can we create variable for it instead?

Reply:
moved: https://github.com/sparkletown/sparkle/pull/2931/commits/493bfda6c66a90eb3c85efbc5943a9e57dacef93